### PR TITLE
Package distribution

### PR DESCRIPTION
**Pull request by achains**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+
+project = u"evops"
+copyright = u"2021, Dmitriy Jarosh"
```
Comment (DmiitriyJarosh) [inline docs/conf.py]: I think its better to mention Pavel here)

File: src/evops/metrics/metrics.py [hunk 2]
```diff
@@ -1,13 +1,26 @@
...
+from evops.metrics.DefaultMetrics import __precision, __accuracy, __recall, __fScore
+from evops.metrics.DiceBenchmark import __dice
+from evops.metrics.IoUBenchmark import __iou
+from evops.metrics.MultiValueBenchmark import __multi_value_benchmark
+from evops.metrics.Mean import __mean
+
+import numpy as np
 
 
 UNSEGMENTED_LABEL = 0
```
Comment (DmiitriyJarosh) [inline src/evops/metrics/metrics.py]: What about this variable, sometimes it causes cyclic imports, better to move to some constants file?

**Review by DmiitriyJarosh — APPROVED** [review]

File: src/evops/metrics/metrics.py (hunk 2 above)
Comment (achains) [reply src/evops/metrics/metrics.py]: Moved to constants.py

File: docs/conf.py (hunk 1 above)
Comment (achains) [reply docs/conf.py]: Sure, done

**Review by pmokeev — APPROVED** [review]
LGTM!

**Review by DmiitriyJarosh — APPROVED** [review]

**Pull request merged**